spoc/contacts.py

```diff
@@ -40,6 +61,67 @@ def __init__(
         self.binary_labels_equal = binary_labels_equal
         self.symmetry_flipped = symmetry_flipped
 
+    @staticmethod
+    def from_uri(uri, mode="pandas"):
```

Comment:
The workflow now seems to look like that: if a user wants to load a table with parameters, they use `FileManager`, but if with URI -- they use `Contacts`/`Pixels`. That's a bit inconsistent: parts of IO API are exposed as Contacts or Pixels API. Should it be changed so that URI is accepted by respective `FileManager` methods?

Reply:
We will move the logic mor the Filemanager but keep direct acces at the Pixel/Contact level.